packages/core/solid/use-chat.ts

```diff
@@ -84,7 +85,18 @@ export function useChat({
   body,
   streamMode,
   generateId = generateIdFunc,
-}: UseChatOptions = {}): UseChatHelpers {
+}: Omit<UseChatOptions, 'api'> & {
+  api?: string;
```

Comment:
This is React specific and can be reverted (no omit `api` and resetting it)

Reply:
reverted in https://github.com/vercel/ai/pull/1669/commits/e150d8a3d3c70ad99d070cd7aed76b765885f1c4